### DISCUSSION TITLE
求助，可不可以注入油猴脚本

### DISCUSSION THREAD

**Discussion by likemuuxi** [tw93/Pake#423] [Q&A]
想打包讯飞星火大模型，原网页带有水印，想加入去水印的油猴脚本，本人不擅长开发，请问能否实现，有无相关教程或案例学习🥰

**Comment by Tlntin** [comment]
参考这几个wike: [高级用法](https://github.com/tw93/Pake/wiki/Pake-%E7%9A%84%E9%AB%98%E7%BA%A7%E7%94%A8%E6%B3%95)
[编译说明](https://github.com/tw93/Pake#development) 
[环境搭建](https://github.com/tw93/Pake/tree/master/bin#windowslinux-%E6%B3%A8%E6%84%8F%E7%82%B9)
